MediaBrowser.Providers/Manager/ProviderManager.cs

```diff
@@ -158,6 +158,14 @@ public async Task SaveImage(BaseItem item, string url, ImageType type, int? imag
             var httpClient = _httpClientFactory.CreateClient(NamedClient.Default);
             using var response = await httpClient.GetAsync(url, cancellationToken).ConfigureAwait(false);
 
+            if (response.StatusCode != HttpStatusCode.OK)
+            {
+                throw new HttpException($"Invalid image received ({response.StatusCode}).")
```

Comment:
Seems a little redundant to include status code in the message when it's available in the `StatusCode`.
```suggestion
                throw new HttpException($"Invalid image received.")
```